site/react/useLogout.md

```diff
@@ -0,0 +1,39 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: useLogout
+  - - meta
+    - name: description
+      content: An overview of the useLogout hook
+  - - meta
+    - property: og:description
+      content: An overview of the useLogout hook
+  - - meta
+    - name: twitter:title
+      content: useLogout
+  - - meta
+    - name: twitter:description
+      content: An overview of the useLogout hook
+---
+
+# useLogout
+
+The `useLogout` hook logs a user out of their Embedded Account on your application.
+
+## Import
+
+```ts
+import { useLogout } from "@alchemy/aa-alchemy/react";
+```
+
+## Usage
```

Comment:
we need a `## Params` section as well to highlight the optional MutationFn params. We can just link out to React Query for that. 

To avoid duplication of code in all the hooks, you can use a `.md` snippet as per https://vitepress.dev/guide/markdown#markdown-file-inclusion